PIL/ImtImagePlugin.py

```diff
@@ -69,7 +69,7 @@ def _open(self):
                 s = s + self.fp.readline()
                 if len(s) == 1 or len(s) > 100:
                     break
-                if s[0] == b"*":
+                if s[0] == ord(b"*"):
```

Comment:
Hang on, found a file and Pillow can open and show it: http://www.realtimerendering.com/resources/GraphicsGems/gemsiv/mrsfoley.im

> * copyright (c) by Alan Paeth, 1993
> * general redistribution for public, non-profit use is granted by the author

And a bit of info [here](https://groups.google.com/forum/#!search/width$20height$20%22pixel$20n8%22/comp.graphics/VXyVCT5i8pQ/MiUhnFcpK_8J).